### ISSUE TITLE
Error using DDP for parameters that do not need to update gradients

### ISSUE THREAD

**Issue by wytcsuch** [pytorch/pytorch#45325]
I use DDP to train with multi GPU.
`self.netF = networks.define_F(opt, use_bn=False).to(self.device)‘
’print('num_gpu',opt['num_gpu'])‘
’if opt['dist']:‘
     ’self.netF = DistributedDataParallel(self.netF,device_ids=[torch.cuda.current_device()],find_unused_parameters = True)‘
’ else:‘
     ’self.netF = DataParallel(self.netF)`
